### PR TITLE
Readme: Fix typos

### PR DESCRIPTION
**Pull request by mbelluzzo**
Signed-off-by: Murilo Belluzzo murilo.belluzzo@intel.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,11 +73,11 @@ already Airborne.
 
 When Airborne, the vehicle follows the transparent sphere (target).
 
-## Throubleshooting ##
+## Troubleshooting ##
 
 Gazebo might fail to open if the needed models or plugins are not found. If
 that happens, make sure the needed plugins have been successfully built and
 installed into your shared libraries directory and that all the needed models
 are present into ~/.gazebo/models. The Gazebo --verbose terminal output is
```
Comment (guiccbr) [inline README.md]: We're not installing the models into ~ anymore.  I suggest changing it to: "... are present into **Gazebo's model path**."

Edit: "... are present into **GAZEBO_MODEL_PATH**. The Gazebo --verbose terminal output ..."

File: README.md [hunk 2]
```diff
@@ -73,11 +73,11 @@ already Airborne.
...
-useful for determining wether a plugin or a models has not not been found.
+useful for determining either a plugin or a models has not not been found.
```
Comment (guiccbr) [inline README.md]: "useful for determining **whether** a plugin or a **model** has not been found" **?**

**Review by guiccbr — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -28,13 +28,12 @@ once with:
 3. Download Gazebo models if necessary:
 
     Gazebo automatically searches and downloads models referenced in the .world
-    file that are not already present in ~/.gazebo/models or in
-    GAZEBO_MODEL_PATH. However, in some cases the download might take a long
+    file that are not already present into GAZEBO_MODEL_PATH.
+    However, in some cases the download might take a long
```
Comment (guiccbr) [inline README.md:32]: +1

File: README.md (hunk 1 above)
Comment (guiccbr) [reply README.md]: Still need to fix this line.

**Review by guiccbr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mbelluzzo) [reply README.md]: gazebo/models..... gazebo/modes EVERYWHERE!

**Review by guiccbr — APPROVED** [review]

**Pull request merged**